packages/nimble-components/src/table-column/text/template.ts

```diff
@@ -7,6 +7,9 @@ TableCellState<TableColumnTextCellRecord, TableColumnTextColumnConfig>
 >`
     <span
         class="${x => (typeof x.cellRecord.value === 'string' ? '' : 'placeholder')}"
+        title="${x => (typeof x.cellRecord.value === 'string'
```

Comment:
Leaving a comment here to capture the discussion from team meeting about whether it's possible to conditionally add a title only if the text is being truncated.

Reply:
Marking await to see more on that topic 👍

Reply:
I'm now setting `mouseover` and `mouseout` handlers that conditionally add and remove the `title` attribute based on whether the text is ellipsized. The way I detect this is by comparing `offsetWidth` (visible width) to `scrollWidth` (entire scrollable width).